pom.xml

```diff
@@ -0,0 +1,391 @@
+        <checkstyle.excludes></checkstyle.excludes>
+        <jacoco.maven.version>0.8.7</jacoco.maven.version>
+        <jakarta.annotation.version>1.3.5</jakarta.annotation.version>
+        <jakarta.enterprise.cdi.version>2.0.1</jakarta.enterprise.cdi.version>
+        <jakarta.json.bind.version>1.0.2</jakarta.json.bind.version>
+        <jakarta.json.version>1.1.6</jakarta.json.version>
+        <jakarta.data.version>${project.version}</jakarta.data.version>
+        <jakarta.validation.version>2.0.1</jakarta.validation.version>
+        <junit.version>5.8.2</junit.version>
+        <maven-javadoc-plugin.vesion>3.3.2</maven-javadoc-plugin.vesion>
+        <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
+        <maven.checkstyle.plugin.version>3.1.2</maven.checkstyle.plugin.version>
+        <maven.compile.version>3.8.1</maven.compile.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.surefire.plugin.version>2.22.2</maven.surefire.plugin.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <sonar.jacoco.reportPath>../target/jacoco.exec</sonar.jacoco.reportPath>
+        <sonar.maven.version>3.3.0.603</sonar.maven.version>
+        <mockito.verson>4.6.0</mockito.verson>
+        <commons.io.version>2.11.0</commons.io.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.verson}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>${mockito.verson}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>${commons.io.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven.compile.version}</version>
+                <configuration>
+                    <target>${maven.compiler.target}</target>
```

Comment:
With properties of proper names - there is no need for this `configuration` section here.

Reply:
Thank you